docs/tools/terraform/reference/cookbook/postgresql-read-replica-recipe.rst

```diff
@@ -0,0 +1,105 @@
+Create a MySQL/PostgreSQL® read replica using Terraform
```

Comment:
Isn't the title a bit misleading? This is talking about PostgreSQL, we can have a hint about being the same for Mysql but the title should reflect what is in there?

Reply:
Also the other titles don't have the "Create a" prefix

Reply:
Thanks Francesco. Valid point. I was using the same title from the Help article but will remove the MySQL reference and "Create a" prefix.